data_ingestion/requirements.txt

```diff
@@ -2,17 +2,17 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile requirements.in
+#    pip-compile data_ingestion/requirements.in
```

Comment:
I think this file should be reverted. It is already inside the data_ingestion/ directory so I don't think it makes sense to add it to the path.